numpy/polynomial/_polybase.py

```diff
@@ -324,9 +324,15 @@ def __format__(self, fmt_str):
         return self._generate_string(self._str_term_unicode)
 
     def __str__(self):
+        scaled_series = not np.all(self.domain == self.window)
```

Comment:
I'd just move this to the `if` statement so that it is easy to see what the condition is without searching through the code.